otio_openassetio/operations/openassetio_media_linker.py

```diff
@@ -7,7 +7,7 @@
 
 from collections import namedtuple
 
-from openassetio import log, exceptions
+from openassetio import log, errors
```

Comment:
[Fix] Be good to add a `hasCapability` check at L165

Reply:
Little bit of linting in 1181013b then added the check in 5281e963

Reply:
Sorry, floating comment [here](https://github.com/OpenAssetIO/otio-openassetio/commit/5281e963b4285ef34c371405f0fe044dc2e3b193#r130680493)

Reply:
^ updated exception in 3b929a98